js/packages/cli/src/commands/upload.ts

```diff
@@ -47,21 +52,33 @@ export async function upload(
 
   const seen = {};
   const newFiles = [];
+  const EXTENSION_ARRAY = [EXTENSION_PNG, EXTENSION_GIF, EXTENSION_JPG];
 
   files.forEach(f => {
-    if (!seen[f.replace(EXTENSION_PNG, '').split('/').pop()]) {
-      seen[f.replace(EXTENSION_PNG, '').split('/').pop()] = true;
+    const extensionRegex = new RegExp(
+      `/(${EXTENSION_PNG})|(${EXTENSION_GIF})|(${EXTENSION_JPG})/g`,
```

Comment:
are file names guaranteed to always only ever be a numbers? e.g. 0.png, 1337.gif If that were not true, this regexp could butcher filenames like my.jpgood-image.jpg